server/hedley/modules/custom/hedley_reports/scripts/recalculate-large-datasets.php

```diff
@@ -20,18 +20,14 @@
   return;
 }
 
-// Get allowed memory limit.
-$memory_limit = drush_get_option('memory_limit', 240);
-
-
 drush_print("Running calculation for Global scope.");
 $duration = create_or_update_results_data_node('global', NULL, NULL, NULL);
 drush_print("Calculation completed within $duration seconds.");
 
 // Resolving unique provinces as they appear at DB.
 $query = db_select('field_data_field_province', 'fp')
   ->fields('fp', array('field_province_value'))
-  ->distinct(TRUE);
```

Comment:
Was it removed to fix a bug or to optimize performance?

Reply:
@AronNovak TRUE is default:
![Selection_628](https://github.com/TIP-Global-Health/eheza-app/assets/17882245/071878d1-c529-4b6d-a37e-9471852cce18)